files/default/neutron-ha-tool.py

```diff
@@ -81,6 +81,9 @@ def parse_args():
                     help='Replicate DHCP configuration to all agents')
     ap.add_argument('--now', action='store_true', default=False,
                     help='Migrate Routers immediately without a delay.')
+    ap.add_argument('--wait-for-ports', action='store_true', default=False,
```

Comment:
> OTOH if we switch the default it seems to me (maybe I am wrong here) we would need to ship an update to the barclamp together with it. AFAIU when this feature is enable we need some config changes (timeouts at least) on the pacemaker resource.

@rhafer The `op start timeout` is already way too low as per [bsc#1011655 (neutron-ha-tool 120s start timeout too low for large clouds)](https://bugzilla.suse.com/show_bug.cgi?id=1011655), so we need an update to `crowbar-ha` anyway.